_notes/launching.md

```diff
@@ -0,0 +1,50 @@
+##Launching Jarvis in Slack
+
+###Create a Testing HTTP Server With ngrok.io 
```

Comment:
For headers, put a space after the hashtags before the title starts. Otherwise the markdown syntax doesn't recognise it. E.g:
```
### Create a Testing HTTP Server With ngrok.io 
```